### PR TITLE
Run gitpod init scripts in parallel

### PR DESCRIPTION
**Pull request by DukeManh**
## Issue This PR Addresses
This makes gitpod run workspace init scripts in parallel (pnpm install + docker build + docker pull) because one doesn't depend on the result of another, and thus saves us some time.

## Type of Change

- [ ] **Bugfix**: Change which fixes an issue
- [x] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [ ] **UI**: Change which improves UI

## Checklist

- [ ] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

**Review by tpmai22 — APPROVED** [review]

**Comment by cindyledev** [comment]
Apparently, pnpm is bundled by default https://github.com/Seneca-CDOT/telescope/issues/2864

File: .gitpod.yml [hunk 1]
```diff
@@ -22,8 +22,9 @@ tasks:
           -e  "s@development\.yml@gitpod\.yml@" \
           config/env.development > .env
     init: |
-      pnpm install
-      docker-compose --env-file .env build $SERVICES
+      pnpm install & \
+      docker-compose --env-file .env build $SERVICES & \
```
Comment (humphd) [inline .gitpod.yml:26]: I wonder if we'll have any race conditions with this (thinking of the pnpm lock file and dependencies service).  cc @JerryHue

Comment (DukeManh) [reply .gitpod.yml:26]: It should be ok as the dep collection script is run on `pre-version`. 
What do u think @JerryHue?

**Comment by DukeManh** [comment]
@cindyledev that's unrelated but I gotchu. #2864

**Comment by cindyledev** [comment]
> @cindyledev that's unrelated but I gotchu. #2864

Thanks SupaDuc, I checked it out

**Review by humphd — DISMISSED** [review]
Based on @JerryHue's points on the call today, this seems like it should be OK.  Let's try it.

**Review by cindyledev — APPROVED** [review]

**Comment by menghif** [comment]
Rebased, ready to merge. @TueeNguyen

**Review by TueeNguyen — APPROVED** [review]

**Pull request merged**